frontend/src/components/MainComponent/MainBody.css

```diff
@@ -9,16 +9,24 @@
 
 .centre-area {
 	display: flex;
-	flex-direction: column;
+	flex-direction: column-reverse;
```

Comment:
I flicked this on and off and I don't believe it's doing anything? Which would make sense given it has only one child

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/3813ca6a-09e6-4dbc-a6c6-a1ad9641afac)

Reply:
In fact, I've checked the other properties and the same argument applies. This class can simply be
```css
.centre-area {
  width:55%;
  background-color: var(--main-text-colour);
}
```